tests/function/gtEvent.cpp

```diff
@@ -301,3 +305,139 @@ TEST_F(LibopaecEventFCommonMOCKHW, event_drv_06) {
   EXPECT_EQ(FPGA_OK, fpgaUnregisterEvent(m_AFUHandle, FPGA_EVENT_POWER_THERMAL,
                                          m_EventHandles[0]));
 }
+
+class LibopaecEventFCommonMOCK : public LibopaecEventFCommonMOCKHW {
+ protected:
+  virtual void SetUp() {
+    MOCK_enable_irq(true);
```

Comment:
Will these tests be totally different from those that use fpgad? If not, can we combine them to be parameterized?

Reply:
Yes, they are significantly different to warrant a different fixture - they can't be run on true hardware, because we cannot easily force the driver to signal an interrupt event.